Makefile

```diff
@@ -201,4 +201,4 @@ release:
 test-e2e: ## Runs e2e tests.
 .PHONY: test-e2e
 test-e2e: deploy
-	go test -mod=readonly -timeout 20m -count 1 -v -tags e2e -run "$(TEST)" ./test/e2e
+	go test -mod=readonly -timeout 40m -count 1 -v -tags e2e -run "$(TEST)" ./test/e2e
```

Comment:
@sunjayBhatia are you seeing e2e taking ~40m to complete?

Reply:
yeah, with the bump to the 1.21.0 kind image, its taking quite a while to finish, timing out deleting namespaces, taking 300s per test sometimes, if you see some of the failed runs

Reply:
its a bit better on my local machine but still really flaky, it seems to be hinged on namespace deletion taking forever

Reply:
even after there are no resources left in a namespace, I see this for a minute or so:

```
apiVersion: v1
kind: Namespace
metadata:
  creationTimestamp: "2021-04-29T18:34:46Z"
  deletionTimestamp: "2021-04-29T18:35:16Z"
  labels:
    contour.operator.projectcontour.io/owning-contour-name: test-default-contour
    contour.operator.projectcontour.io/owning-contour-namespace: contour-operator
    kubernetes.io/metadata.name: projectcontour
  name: projectcontour
  resourceVersion: "1629"
  uid: dfb0e8af-d7b5-4868-8351-0df1319bff71
spec:
  finalizers:
  - kubernetes
status:
  conditions:
  - lastTransitionTime: "2021-04-29T18:35:22Z"
    message: All resources successfully discovered
    reason: ResourcesDiscovered
    status: "False"
    type: NamespaceDeletionDiscoveryFailure
  - lastTransitionTime: "2021-04-29T18:35:22Z"
    message: All legacy kube types successfully parsed
    reason: ParsedGroupVersions
    status: "False"
    type: NamespaceDeletionGroupVersionParsingFailure
  - lastTransitionTime: "2021-04-29T18:35:22Z"
    message: All content successfully deleted, may be waiting on finalization
    reason: ContentDeleted
    status: "False"
    type: NamespaceDeletionContentFailure
  - lastTransitionTime: "2021-04-29T18:35:22Z"
    message: 'Some resources are remaining: pods. has 1 resource instances'
    reason: SomeResourcesRemain
    status: "True"
    type: NamespaceContentRemaining
  - lastTransitionTime: "2021-04-29T18:35:22Z"
    message: All content-preserving finalizers finished
    reason: ContentHasNoFinalizers
    status: "False"
    type: NamespaceFinalizersRemaining
  phase: Terminating
```

Reply:
similar status updates occur with the 1.20 kind image, but namespace deletion is much faster, on the order of seconds not minutes